sources/core/objects/mobs/Human.cpp

```diff
@@ -427,6 +429,14 @@ void Human::Live()
                 }
             }
         }
+        if(plasma > 35)
```

Comment:
It is wise to avoid magic numbers in code (they still appear time from time in old code, but it will be fixed eventually).
So here it is needed to make some constant for `35`, e.g.:
```cpp
const int PLASMA_TOXINS_THRESHOLD = 35;
if (plasma > PLASMA_TOXINS_THRESHOLD)
```

Reply:
Also codestyle requests space after `if`:
```cpp
if (something)
```